lambda-notes/src/components/ViewNote.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Sidebar from './Sidebar';
+import Note from './Note';
+
+const ViewNote = (props) => {
+  console.log('ViewNote', props);
+  const selectedId = parseInt(props.match.params.id, 10)
```

Comment:
I like what you did here.  alternatively you could have used type coercion.  so ==   I.E. 10 === '10' is false but 10 == '10' is true.  Good work!